docs/app/pages/components/components-sections/input-controls/radio-button/radio-button.component.html

```diff
@@ -71,16 +71,16 @@
     </tr>
 </uxd-api-properties>
 
-<uxd-api-properties tableTitle="Outputs">
+<uxd-api-properties tableTitle="Radio Button Outputs">
     <tr uxd-api-property name="valueChange" type="any">
         Emits when the value has been changed.
     </tr>
 </uxd-api-properties>
 
 <uxd-api-properties tableTitle="uxRadioButtonGroup Inputs">
     <tr uxd-api-property name="value" type="any">
-        This should be a two way binding and will store the currently selected option. Each radio button in the same
-        group should have the same <code>value</code> variable.
+        This should be a two way binding and will store the currently selected option. Only the top level
```

Comment:
I'm not sure this sentence: "Only the top level element should have a <code>value</code> variable." is needed at all. Our example has been updated to show how this should be used and if they decided to also add it to the radio buttons it would just be duplicate code, it should still work as expected. If we really did want to explicitly state this it should be clearer, indicating that the if the radio button group has a `value` binding then the radio buttons do not require them rather than "top level element".